.github/workflows/automerge.yml

```diff
@@ -25,5 +26,18 @@ jobs:
           MERGE_METHOD: squash
           MERGE_COMMIT_MESSAGE: "pull-request-title"
           MERGE_RETRIES: 3
-          MERGE_RETRY_SLEEP: 10000
+          MERGE_RETRY_SLEEP: 50000
+          UPDATE_METHOD: squash
+
+      - name: Attempt merge a release PR
```

Comment:
@kylebuch8 I added this so we can use the automerge functionality on release pull requests and it will use the merge approach instead of the squash.  Just need to make sure release PRs have the `release` label and that their title is the commit message you want to use.